core/src/wallet/ripple/api_impl/RippleLikeTransactionApi.h

```diff
@@ -72,6 +72,8 @@ namespace ledger {
             std::vector<api::RippleLikeMemo> getMemos() override;
             void addMemo(api::RippleLikeMemo const& memo) override;
             std::experimental::optional<int64_t> getDestinationTag() override;
+            std::string getStatus() override;
```

Comment:
Yep, I’ve thought about it. And Djinni. Currently, I don’t want to apply `constness` because if I do it here, it will be inconsistent with the rest of the API (which doesn’t use it). I agree with your concern and it should be fixed. I think we already have a Jira ticket open about that matter in the `Housekeeping` epic. :)